src/js/stores/GroupsStore.js

```diff
@@ -0,0 +1,36 @@
+var EventEmitter = require("events").EventEmitter;
+
+var AppDispatcher = require("../AppDispatcher");
+var GroupsEvents = require("../events/GroupsEvents");
+
+var Util = require("../helpers/Util");
+
+var GroupsStore = Util.extendObject(EventEmitter.prototype, {});
+
+AppDispatcher.register(function (action) {
+  switch (action.actionType) {
+    case GroupsEvents.SCALE_SUCCESS:
+      GroupsStore.emit(GroupsEvents.SCALE_SUCCESS);
```

Comment:
We don't need it yet.
Against the rule "don't add it, if you don't need it right away".